deprecations/README.md

```diff
@@ -1,65 +1,88 @@
+Such features are normally deprecated because they are superseded by a better alternative. They can include object properties, entire objects or entire API operations.
+The list of deprecations is as follows. Individual items are also highlighted in the [Changelog](../changelog/README.md) when they occur.
+Historic deprecations that have already been discontinued may not be listed.
 
-These are the operations that we will deprecate in 2021. While we are not yet enforcing these changes, and are yet to finalise a date for these deprecations, we strongly advise you to review these and if you are using them in your API calls, to update them accordingly.
+> **Important:** We strongly advise you to review this list and if you are using any of the deprecated items in your integration, to update your implementation accordingly.
 
-### Deprecated operations
+The table columns have the following meanings:
 
-Following legacy operations to be completely removed:
-* `companies/getAllByName` (replaced by [Get all companies](../operations/companies.md#get-all-companies))
-* `bills/getAllByIds` (replaced by [Get all bills](../operations/bills.md#get-all-bills))
-* `bills/getAllByCustomers` (replaced by [Get all bills](../operations/bills.md#get-all-bills))
-* `bills/getAllClosed` (replaced by [Get all bills](../operations/bills.md#get-all-bills))
-* `creditCards/getAllByIds` (replaced by [Get all credit cards](../operations/creditcards.md#get-all-credit-cards))
-* `creditCards/getAllByCustomers` (replaced by [Get all credit cards](../operations/creditcards.md#get-all-credit-cards))
-* `reservations/getAllByIds` (replaced by [Get all reservations](../operations/reservations.md#get-all-reservations))
-* `reservations/getAllByCustomers` (replaced by [Get all reservations](../operations/reservations.md#get-all-reservations))
-* `reservations/getAllByNumbers` (replaced by [Get all reservations](../operations/reservations.md#get-all-reservations))
-* `customers/getAllByIds` (replaced by [Get all customers](../operations/customers.md#get-all-customers))
-* `customers/getAllByEmails` (replaced by [Get all customers](../operations/customers.md#get-all-customers))
-* `customers/getAllByName` (replaced by [Get all customers](../operations/customers.md#get-all-customers))
+* __Feature__ - What entity or feature is being deprecated
+* __Comments__ - Additional information, such as the reason for the deprecation and what the replacement is
+* __Deprecated__ - The date at which the deprecation notice was given (see the [Changelog](../changelog/README.md))
+* __Discontinued__ \- The date at which it is planned to discontinue support completely; a value of '-' indicates no date has been set
 
-### Deprecated time filters 
+## Deprecated operations
+
+| Feature | Comments | Deprecated | Discontinued |
+| :-- | :-- | :-- | :-- |
+| `companies/getAllByName` | Replaced by [Get all companies](../operations/companies.md#get-all-companies) | ? | - |
```

Comment:
I'd suggest marking those deprecated at 2020-01-01... It's pretty difficult to find when we really marked them as deprecated.

Reply:
@MikeAdamsMews we covered this also during our meeting, that we can go with one unified date. Let's do it :)